package.json

```
@@ -4,6 +4,7 @@
   "private": true,
   "homepage": "/",
   "scripts": {
+    "analyze": "source-map-explorer 'build/static/js/*.js'",
```

```suggestion
    "analyze": "source-map-explorer 'build/static/js/*.js' --html > build/stats.html",
```
Pour éviter qu'il ne se retrouve sur le dépôt.